src/main/groovy/org/edx/jenkins/dsl/debug/XmlOutputRunner.groovy

```diff
@@ -0,0 +1,72 @@
+package org.edx.jenkins.dsl.debug
+
+import groovy.io.FileType
+import javaposse.jobdsl.dsl.*
+
+/**
+ * Runs the DSL engine and writes the XML output to files.
+ */
+
+String pattern = System.getProperty('pattern')
+
+if (!pattern) {
+    println 'usage: -Dpattern=<pattern>'
+    System.exit 1
+}
+
+File outputDir = new File('./build/debug-xml')
+outputDir.deleteDir()
```

Comment:
This confuses me....shouldn't that be picked up by clean as well? Or, if `clean` isn't clean enough...should this fall in some sort of `setUp` function?